# %YAML:1.0
# ---
image_width: 640
image_height: 480
camera_matrix: # !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 462.139, 0.0, 320.0, 0.0, 462.139, 240.0, 0.0, 0.0, 1.0 ]
distortion_coefficients: #  !!opencv-matrix
   rows: 5
   cols: 1
   dt: d
   data: [ 0.0, 0.0, 0.0, 0.0, 0.0 ]
